components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -137,6 +137,8 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
     private var autoCompleteSpans: List<Any>? = null
     // Do not process autocomplete result
     private var discardAutoCompleteResult: Boolean = false
+    // Controls whether or not suggestions are shown
+    var enableAutocomplete: Boolean = true
```

Comment:
It looks like you are not using this and instead rely on `isEnabled`?

Reply:
Ah, good catch. This was leftover from a previous implementation.